examples/cpp-simple/server.cc

```diff
@@ -16,101 +16,107 @@
 // This example really does nothing other than show how to integrate
 // the C++ SDK.
 
-#include <iostream>
-#include <thread>
 #include <agones/sdk.h>
 #include <grpc++/grpc++.h>
+#include <iostream>
+#include <thread>
 
 // send health check pings
-void doHealth(agones::SDK *sdk) {
-    while (true) {
-        if (!sdk->Health()) {
-            std::cout << "Health ping failed" << std::endl;
-        } else {
-            std::cout << "Health ping sent" << std::endl;
-        }
-        std::this_thread::sleep_for(std::chrono::seconds(2));
+void DoHealth(std::shared_ptr<agones::SDK> sdk) {
+  while (true) {
+    if (!sdk->Health()) {
+      std::cout << "Health ping failed\n";
+    } else {
+      std::cout << "Health ping sent\n";
```

Comment:
(optional) You might consider writing this like:

```cc
while (true) {
  book ok = sdk->Health();
  std::cout << "Health ping " << (ok ? "sent" : "failed) << "\n";
  std::this_thread::sleep_for(...);
}
```

Or similar. The reason being that often times less nesting results in more readable code.

Reply:
done.